src/main/java/pl/wojciechkostecki/revel/exception/BadRequestException.java

```diff
@@ -0,0 +1,9 @@
+package pl.wojciechkostecki.revel.exception;
+
+import java.text.MessageFormat;
+
+public class BadRequestException extends RuntimeException{
+    public BadRequestException(final  Long menuId, final Long localId){
+        super(MessageFormat.format("Menu: {0} is already assigned to local: {1}", menuId,localId));
```

Comment:
Name of the exception is generic, but you ruined it with the constructor that is only usable for Local/Menu. Your exception should accept message in string and that's all. This way it can be used throughout the app. :)